nfcd/src/main/jni/src/helper/EEManager.cpp

```diff
@@ -46,9 +46,9 @@ std::set<uint16_t> EEManager::findActiveEEs() const {
             LOGD("EE found: %x with {status=%d, ee_interface=[%s], la_protocol=%d, lb_protocol=%d, lf_protocol=%d}",
                  ee_handle, ee_status, str_ee_interfaces.c_str(), la_protocol, lb_protocol, lf_protocol);
 
-            // only non-deactivated proprietary EEs
+            // only non-deactivated proprietary EEs or EEs without an interface
             auto itProp = std::find(ee_interface.begin(), ee_interface.end(), NCI_NFCEE_INTERFACE_PROPRIETARY);
-            if (ee_status != NFA_EE_STATUS_INACTIVE && itProp != ee_interface.end())
+            if (ee_status != NFA_EE_STATUS_INACTIVE && (itProp != ee_interface.end() || ee_interface.empty()))
```

Comment:
Second part of the condition should probably be extracted to variable for readability purposes.